library/swift/src/Client.swift

```diff
@@ -9,23 +9,28 @@ public protocol Client {
   /// - parameter handler: Handler for receiving stream events.
   ///
   /// - returns: Emitter for sending streaming data outward.
-  func startStream(with request: Request, handler: ResponseHandler) -> StreamEmitter
-
-  /// Convenience function for sending a unary request.
-  ///
-  /// - parameter request:  The request to send.
-  /// - parameter body:     Serialized data to send as the body of the request.
-  /// - parameter trailers: Trailers to send with the request.
-  func sendUnary(_ request: Request, body: Data?,
-                 trailers: [String: [String]], handler: ResponseHandler)
+  func send(_ request: Request, handler: ResponseHandler) -> StreamEmitter
 }
 
 extension Client {
-  /// Convenience function for sending a unary request without trailers.
+  /// Convenience function for sending and completing a non-streaming request.
```

Comment:
nit/minor suggestion:
`/// Convenience function for sending a complete (non-streamed) request.`

Reply:
updated